both/methods/read/channels.js

```diff
@@ -0,0 +1,35 @@
+Meteor.methods({
+  /*
+   * Reads a Channel and searches for its images
+   *
+   * @param {Channel} the Channel object to read
+   * @return {Images} an array of image objects
+   */
+  '/channels/': function(channel) {
+    // Check Arguments
+    check(channel, Channel);
+
+    // Check if the user is logged in? Not sure this is necessary
+    // if(!Meteor.user()) throw new Meteor.Error('logged-out','You must be logged in to create a channel.');
+    
+    if (channel.validate()) {
+      return Modules.both.getImages(channel);
+    } else {
+      channel.throwValidationsException();
+    }
+  },
+  
+  getUrls: function(channel) {
+    // Check Arguments
+    check(channel, Channel);
+
+    // Check if the user is logged in? Not sure this is necessary
+    // if(!Meteor.user()) throw new Meteor.Error('logged-out','You must be logged in to create a channel.');
+    
+    if (channel.validate()) {
+      return Modules.both.getUrls(channel);
+    } else {
+      channel.throwValidationsException();
+    }
+  }
+});
```

Comment:
I think I need to redo this because I wasn't thinking about the endpoint. I either need to make another Meteor method so the client can call it or rewrite these two. Right now it takes a Channel object in as a parameter but it should just take in some JSON object that has the channel data, i.e. title and query because I'm not sure we can send Astronomy classes through the DDP.